6_visualize/src/create_gif_tasks.R

```diff
@@ -0,0 +1,54 @@
+# create task table for gif generation
+
+
+
+create_gif_tasks <- function(timestep_ind, folders, view_ind, basemap_ind, storm_line_ind, cfg){
+
+  timestep <- readRDS(sc_retrieve(timestep_ind))
+
+  cfgs <- c('a','b','c') # dummy placement for different configurations; will eventually be configurations that hold information about size, aspect, ect...
+
+  tasks <- tidyr::crossing(timestep, cfgs) %>%
+    unite(task_name, cfgs, timestep, sep = '_', remove = F) %>%
+    mutate(task_name = gsub(' ', '_', task_name),
+           date_hour = strftime(timestep, format = '%Y%m%d_%H', tz = 'UTC'))
+
+  # function to sprintf a bunch of key-value (string-variableVector) pairs, then
+  # paste them together with a good separator for constructing remake recipes
+  psprintf <- function(..., sep='\n      ') {
+    args <- list(...)
+    strs <- mapply(function(string, variables) {
+      spargs <- if(string == '') list(variables) else c(list(string), as.list(variables))
+      do.call(sprintf, spargs)
+    }, string=names(args), variables=args)
+    paste(strs, collapse=sep)
+  }
```

Comment:
are you liking this function? should we formalize it in `scipiper` one of these days?

Reply:
yeah! it's very handy for the task plan creation and I always copy / paste it as is anyways

Reply:
I can create an issue in scipiper